docs/3.0.0-beta.x/plugins/users-permissions.md

```diff
@@ -435,3 +435,6 @@ You can update these template in the **Email Templates** tab in the admin panel.
   - ...and every other fields that you added manually in the model.
 - `CODE` corresponds to the CODE generated to be able confirm the user email.
 - `URL` is the Strapi backend URL that confirm the code (by default `/auth/email-confirmation`).
+
+## Security configuration
+JWT tokens can be verified and trusted because the information is digitally signed. To sign a token a *secret* is required. By default Strapi generates one that is stored in `./your-app/extensions/users-permissions/config/jwt.json`. This is useful during development but for security reasons it's **recommended** to set a custom token via environment variable `JWT_SECRET` when deploying to production.
```

Comment:
`it's **recommended**` Go ahead and break that up and use `it is **recommended**`

Also `via an environment variable`

Reply:
Fixed, thanks 👍